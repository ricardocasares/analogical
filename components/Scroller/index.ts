import styled from "@app/lib/styled";

export const Horizontal = styled.div`
  overflow-x: scroll;
  overflow-y: hidden;
  white-space: nowrap;
  -webkit-overflow-scrolling: touch;

  ::-webkit-scrollbar {
    display: none;
  }
`;

export default Horizontal;
