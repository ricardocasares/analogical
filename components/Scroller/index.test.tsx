import React from "react";
import renderer from "react-test-renderer";
import serializer from "jest-emotion";
import Scroller from "@app/components/Scroller";

expect.addSnapshotSerializer(serializer);

test("renders with correct styles", () => {
  expect(
    renderer
      .create(
        <Scroller>
          <a>hello world</a>
        </Scroller>
      )
      .toJSON()
  ).toMatchSnapshot();
});
