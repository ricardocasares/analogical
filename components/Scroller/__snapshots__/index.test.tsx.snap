// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with correct styles 1`] = `
.emotion-0 {
  overflow-x: scroll;
  overflow-y: hidden;
  white-space: nowrap;
  -webkit-overflow-scrolling: touch;
}

.emotion-0::-webkit-scrollbar {
  display: none;
}

<div
  className="emotion-0"
>
  <a>
    hello world
  </a>
</div>
`;
